weave

```diff
@@ -883,6 +883,24 @@ attach() {
     util_op attach-container $ATTACH_ARGS $CONTAINER $BRIDGE $MTU "$@"
 }
 
+check_not_running_v2_plugin() {
+    if is_docker_plugin_enabled $PLUGIN_V2_NAME; then
+        echo "Docker plugin \"$PLUGIN_V2_NAME\" is already running." >&2
+        return 1
+    fi
+}
+
+# Docker Plugin V2
+is_docker_plugin_enabled() {
+    util_op is-docker-plugin-enabled $PLUGIN_V2_NAME 2>/dev/null
+}
+
+# Docker Plugin (part of weaver process, so V2 or V1).
+# Assumes that plugin is running inside Docker container.
+is_plugin_enabled() {
```

Comment:
Should we actually rename these two functions so that it is clearer which one applies to which plugin?
e.g.:
```
is_plugin_enabled        -> is_plugin_v1_enabled
is_docker_plugin_enabled -> is_plugin_v2_enabled
```